Shared/shared.template.json

```diff
@@ -348,7 +348,7 @@
       }
     },
     {
-      "apiVersion": "2018-03-01",
+      "apiVersion": "2017-05-10",
```

Comment:
This has already been changed. See https://github.com/DFE-Digital/login.dfe.infrastructure/commit/fe62a4f25e0fe91c4e1137b68e1875337f1efcbf

Reply:
I've updated that in https://github.com/DFE-Digital/login.dfe.infrastructure/pull/71/commits/9e08b25895567458df94af5a1ad9f61085ad6b4f. The change in master needs merging in to develop.